### ISSUE TITLE
Could add instructions to create a new branch

### ISSUE THREAD

**Issue by elenajp** [zero-to-mastery/start-here-guidelines#13477]
Hi,

I think it would be a good idea to include instructions to create a new branch and how to create it, so that the user does not work on master. I am new to open source and think this is a great way to practice! Thank you 😀

**Comment by sophiabrandt** [comment]
The problem is that it's already quite complicated for a beginner to grasp the code flow.

I suggest adding these steps in a separate document and link to them in `README.md`, so students can decide if they want to add these steps to their workflow.

**Comment by rajulkoshta** [comment]
i want to work on this can i update it as a link to another readme.md  where instructions to create new branch can be found

**Comment by sophiabrandt** [comment]
@rajulkoshta it sounds to me like @elenajp wanted to tackle this. That's why they opened this issue.

Please clear this up under yourselves.

**Comment by elenajp** [comment]
@sophiabrandt yeah I was also thinking if it is quite complicated for a beginner but then I was also thinking that it would be a good idea to get into the habit of creating new branches, as I have forgotten to do it myself. Great idea about creating these steps in a separate document! 

@rajulkoshta you can work on this if you like 😀 just let me know if you change your mind and I will do it.

Have a great day everyone! ⭐️

**Comment by Anuragkillswitch** [comment]
Can I work on this?

**Comment by elenajp** [comment]
@rajulkoshta wants to work on it, unless they change their mind 🙂

**Comment by Anuragkillswitch** [comment]
> @rajulkoshta wants to work on it, unless they change their mind 🙂

Alright alright

**Comment by rajulkoshta** [comment]
Yaa you can go for it@Anuragkillswitch

**Comment by siddharth329** [comment]
I have created a short instruction about making branch in repository, can I make a pull request?
